### PR TITLE
add business name field, method and test

### PR DESCRIPTION
**Pull request by ros3bud**
@cgrayson does this look alright?

### CODE REVIEW

File: spec/outbound-spec.coffee [hunk 1]
```diff
@@ -243,7 +248,8 @@ describe 'Outbound', ->
           comments: 'nostrum sed rem totam dolore atque non unde'
           credit_score: 'excellent'
           mobile_opt_in: true
-          company: 
+          company:
+            company: 'edu trainers'
```
Comment (cgrayson) [inline spec/outbound-spec.coffee]: It [looks like](https://github.com/activeprospect/leadconduit-fields/blob/master/fields/company.json) just `company` is the full field name, with no prefix.

File: src/outbound.coffee [hunk 2]
```diff
@@ -41,6 +41,7 @@ request = (vars) ->
     average_monthly_sales: parseAverageMonthlySalesId(lead.company.monthly_sales)
     industry: parseIndustryId(lead.company.industry)
     time_to_funding: parseTimeToFundingId(lead.loan.timeframe_days)
+    business_name: parseName(lead.company.company)
```
Comment (cgrayson) [inline src/outbound.coffee]: Since this is simple, and only used once, maybe inline it: `business_name: _s.titleize(lead.company?.trim())`.

**Comment by ros3bud** [comment]
Ok -thanks!

**Comment by ros3bud** [comment]
@cgrayson updated. Look ok?

File: spec/outbound-spec.coffee (hunk 1 above)
Comment (cgrayson) [inline spec/outbound-spec.coffee]: Did you see this note?

Comment (ros3bud) [inline spec/outbound-spec.coffee]: Sorry, I did not until now.

**Pull request closed without merging**